src/main/java/io/temporal/internal/statemachines/ActivityStateMachine.java

```diff
@@ -415,5 +404,6 @@ private void createRequestCancelActivityTaskCommand() {
                 RequestCancelActivityTaskCommandAttributes.newBuilder()
                     .setScheduledEventId(getInitialCommandEventId()))
             .build());
+    parameters = null; // avoid retaining large input for the duration of the activity
```

Comment:
curious what inspired you to make this change (whether it just occurred to you or you saw something interesting during local debugging / execution)

Reply:
I was thinking about it in my spare time but was forgetting about it when getting to the computer :).